Common/Constants.cs

```diff
@@ -5,6 +5,22 @@ namespace Kudu
 {
     public static class Constants
     {
+        //Scan functionality files
+        public const string ScanLockFile = "scan.lock";
+        public static string ScanStatusFile = "status.json";
+        public static string ScanLogFile = "scan_log.log";
+        public static string ScanFolderName = "Scan_";
+        public static string MaxScans = "2";
+        public static string ScanDir = "/home/site/wwwroot";
+        public static string ScriptPath = "/custom_scripts/daily_scan_script.sh";
+        public static string ScanCommand = ScriptPath+" "+ScanDir;
+        public static string ScanTimeOut = "500000";
+        public static string ScanManifest = "modified_times.json";
+        public static string AggregrateScanResults = "aggregrate_scans.log";
+        public static string NoModifiedFiles = "No files modified since last scan";
```

Comment:
There is a 'Resources' file in Kudu.Core - intended to be modified for multi language support. We add all the error messages to that. Constants file is just to store default paths and app settings.

Reply:
Ok got it, will make the changes in the Resources file in Kudu.Services as this code is in the Controller.